docs/Concepts/c_using_demos.md

```diff
@@ -0,0 +1,56 @@
+				obj=HeadJTASTest()
+				obj.add_point(list(current_angles), 0.0)
+				obj.add_point([-0.5,1],5)
+				obj.start()
+		In the code, they use the velocity to compute the time it takes to go to each position. You can also do the same.
+	jaco_action_client.py is an interface to move the arms in angular control.
+		add_point: This method adds an angular position and a time that corresponds to the time from start to be at that position. You can then create trajectories for the arms with a number of positions to go through.
+	torso_action_client.py is an interface to move the torso up and down.
+		add_point: This method gives a linear position for the torso and a time point for when it will be.
+	gripper_action_client.py is an interface to open and close the grippers.
+		command: Commands a position for the grippers. Position 0 corresponds to open position and position 0.165 corresponds to closed position.
+
+
+movo_common/si_utils/src/si_utils/: This folder contains example codes on how to use these classes for your needs.
+
+-------------------------------------------------------------------------------------------------------
+movo_demos:
+	XXX = robot [When you connect to the robot, ROS MASTER is Movo2] and sim [In simulation, ROS_MASTER is your computer] 
+	XXX_map_nav.launch: Opens Rviz and Gazebo to navigate in a map. The map should be added in movo_demos/maps and the XXX_map_nav.launch file. 
+		To use the demo, you click on 2D pose estimate and click on the start point of the map. This will be the initial pose of the robot. 
+		Then, you click on 2D Nav goal to set a goal for the robot. The robot will then naviguate to that position using the map and the sensors of the base. 
+	XXX_mapping.launch: Launch the mapping node. It opens Rviz for you to visualize how the mapping is going. It also launches the teleop node for you to move the robot around and map the surrounding of the robot. The mapping will update on Rviz and you will be able to save it at the end by running this command line : rosrun map_server map_saver -f nameOfFile (This will save the .pgm and .yaml file in your home directory)
+
+	XXX_robot_demo.launch: Launch a simple demo to showcase the robot capabilities.
+	XXX_teleop.launch: Launch the joystick teleop node. You will be able to move the robot around with the joystick connected to the computer or directly with the Movo.
+	XXX_assisted_teleop.launch: Launches the teleop.launch node and uses a footprint area around the robot to avoid collision with surrounding objects.
+	dance_invitation.launch: Movo will track your face and when it recognizes a face it will position itself and ask the user if it wants to dance. If it does, it will extend its right arm. You will 		be able to control Movo by applying forces on the end effector and make it move.
+	face_tracking: The face_tracking will launch and will follow a person's face (after recognizing it). If you move to fast, the head may not follow you.
```

Comment:
"will follow a person's face" maybe just explicitely state the Kinect will follow? The whole movo following someone would be quite creepy